app/controllers/doc_controller.rb

```diff
@@ -65,7 +66,11 @@ def set_doc_version
     return unless @doc_file
     version = params[:version]
     if version
-      @doc_version = @doc_file.doc_versions.for_version(version)
+      if /\d+\.\d+\.\d+/ =~ version
+        @doc_version = @doc_file.doc_versions.for_version(version)
+      else
+        @doc_version = @doc_file.doc_versions.latest_version(version)
```

Comment:
we can probably simply this if/else cascade into a single 

if A OR B then `. for_version ` else `.latest_version ` no?

Reply:
also, latest_version does not receive a version argument, right?

Reply:
for the or, will do.

Here latest_version receives the language.